core/include/gnuradio-4.0/Port.hpp

```diff
@@ -843,6 +843,10 @@ class DynamicPort {
         [[nodiscard]] virtual bool
         updateReaderInternal(InternalPortBuffers buffer_other) noexcept
                 = 0;
+
+        [[nodiscard]] virtual std::string
+        valueTypeName() const
```

Comment:
Do we need this as mandatory public-API, or is this more a UI convenience function to disentangle types?

We need to shrink/clean-up the public-API and I am not sure whether this is needed as part of the Port interface. Stand-alone helper function?

Reply:
We need to be able to tell the type of a dynamic port. Needs to be here as it is a part of type erasure of a typed port.

Reply:
There is an existing `std::any Port<...>::defaultValue()` method.  It's type erased and can be used at the lib and user-end to derive the value type of the port.

Could we use that instead?